test/mpi/test_bcast.cc

```diff
@@ -22,14 +22,6 @@ TestBcast::TestBcast(size_t _msgsize, ucc_test_mpi_inplace_t _inplace,
     root = _root;
     args.coll_type = UCC_COLL_TYPE_BCAST;
 
-    if (TEST_INPLACE == inplace && !ucc_coll_inplace_supported(args.coll_type)) {
-        test_skip = TEST_SKIP_NOT_IMPL_INPLACE;
-    }
-    if (skip_reduce(test_max_size < _msgsize, TEST_SKIP_MEM_LIMIT,
```

Comment:
where is the MEM_LIMIT skip check fo bcast now?

Reply:
fixed